flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -877,15 +880,15 @@ public TimeCharacteristic getStreamTimeCharacteristic() {
 
 	/**
 	 * Reads the given file line-by-line and creates a data stream that contains a string with the contents of each such
-	 * line. Depending on the provided {@link WatchType}, the source may periodically monitor (every {@code interval} ms)
-	 * the path for new data ({@link WatchType#REPROCESS_WITH_APPENDED}), or process once the data currently in the path and
-	 * exit ({@link WatchType#PROCESS_ONCE}).
+	 * line. Depending on the provided {@link ProcessingMode}, the source may periodically monitor (every {@code interval} ms)
+	 * the path for new data ({@link ProcessingMode#PROCESS_CONTINUOUSLY}), or process once the data currently in the path and
+	 * exit ({@link ProcessingMode#PROCESS_ONCE}).
 	 *
 	 * <p>
-	 * <b> NOTES ON CHECKPOINTING: </b> If the {@code watchType} is set to {@link WatchType#PROCESS_ONCE}, the source
-	 * (which executes the {@link FileSplitMonitoringFunction}) monitors the path <b>once</b>, creates the
+	 * <b> NOTES ON CHECKPOINTING: </b> If the {@code watchType} is set to {@link ProcessingMode#PROCESS_ONCE}, the source
+	 * (which executes the {@link ContinuousFileMonitoringFunction}) monitors the path <b>once</b>, creates the
```

Comment:
I think we can remove the "(which executes the {@link ContinuousFileMonitoringFunction})" part. The source does not execute this, it is itself the source.